cmake/GenerateResourceSpec.cmake

```
@@ -20,7 +20,7 @@ if(ROCMINFO_EXIT_CODE)
   message(FATAL_ERROR ${ROCMINFO_STDERR})
 endif()
 
-string(REGEX MATCHALL [[--(gfx[0-9]+)]]
+string(REGEX MATCHALL [[--(gfx[0-9]+a?)]]
```

```suggestion
string(REGEX MATCHALL [[--(gfx[0-9a-f]+)]]
```
You might now better than me in this, but AFAIK, the target ids are hexadecimal. (E.g the [llvm amdpgu page](https://www.llvm.org/docs/AMDGPUUsage.html#processors) also has `gfx90c` listed).
I don't think this file should try to filter for supported architectures by rocm (if that was the intention).

---

Thanks for the correction.  Fixed.